### PR TITLE
latest beta packages; new library api; tooling upgrades

### PR DESCRIPTION
**Pull request by dmosites**
Using new libraries for Registry, Server, SwapERC20, etc.
Using webpack 5 via react-scripts upgrade.
Dependency upgrades and cleanup.
Tested on Goerli with RFQ and OTC swaps.

### CODE REVIEW

File: src/components/OrderDetailWidget/OrderDetailWidget.tsx [hunk 1]
```diff
@@ -134,12 +134,10 @@ const OrderDetailWidget: FC<OrderDetailWidgetProps> = ({ order }) => {
...
-    const errors = await check(
-      order,
-      order.senderWallet,
-      order.chainId,
-      library!.getSigner()
-    );
+    let errors: any[] = [];
+    if (library) {
+      errors = await check(order, order.senderWallet, order.chainId, library);
+    }
```
Comment (piersss) [inline src/components/OrderDetailWidget/OrderDetailWidget.tsx:140]:
```suggestion
    if (!library) {
      return;
    }

    const errors = await check(
      order,
      order.senderWallet,
      order.chainId,
      library,
    );
```

File: src/components/Toasts/TransactionToast.tsx [hunk 2]
```diff
@@ -41,15 +41,15 @@ export type TransactionToastProps = {
   /**
    * Token Info of sender token
    */
-  senderToken?: TokenInfo;
+  senderToken?: TokenInfo | null;
```
Comment (piersss) [inline src/components/Toasts/TransactionToast.tsx:44]: Don't do null and undefined together.

```suggestion
  senderToken?: TokenInfo;
```

In ToastController:
```
<TransactionToast
            onClose={() => toast.dismiss(t.id)}
            type={type}
            transaction={transaction}
            senderToken={senderToken || undefined}
            signerToken={signerToken || undefined}
            error={error}
          />
```

**Comment by piersss** [comment]
OTC swaps work but RFQ not, are the makers empty?

**Comment by dmosites** [comment]
> OTC swaps work but RFQ not, are the makers empty?

It takes some seconds for the test maker to spin up after idling so might try a couple times. Just worked on my end.

**Review by piersss — APPROVED** [review]
Tested again and all works.

**Pull request merged**